metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ghoneycutt-ssh",
-  "version": "3.37.0",
+  "version": "3.38.0",
```

Comment:
don't bump versions in your PR's. I might merge someone else's first, cut a release, then merge this and cut a release and then the version here would be wrong.